pubsubplus/templates/solaceConfigMap.yaml

```diff
@@ -157,15 +158,17 @@ data:
           "Local Active")
             echo "`date` INFO: ${APP}-Node activity status is Local Active, after ${run_time} seconds"
             # We should only be here on new cluster create, if not likely a bug

```

Comment:
For this comment: `# We should only be here on new cluster create, if not likely a bug`
After more testing, I can see that  if we are here on OTHER than new cluster create, it may not be necessary a bug in the broker code, just an indication of not normal HA cluster procedures. E.g.: initiating a failover before redundancy is full up after a prev transition.
So suggesting to modify the comment to: `# We should only be here on new cluster create, if not this is an indication of unexpected HA procedures`